test/api/repositories/template-inputs/001-inputs/expected-response.json

```diff
@@ -25,5 +25,6 @@
         }
       ]
     }
-  ]
+  ],
+  "preconfiguredSteps": []
```

Comment:
Field sa vrati aj pri `use` operacie, je to schvalne, ... teraz bude vzdy prazdny no:
- UI bude mat iba jeden typ response pre `use/upgrade` -> vyhoda.
- Do  buducna mozu byt niektore steps prekonfigurovane aj pri `use`, napr. demo data, alebo SNFK provisioning by Keboola.